### ISSUE TITLE
the disk consuming for raw reads correction

### ISSUE THREAD

**Issue by longzhangnation** [Nextomics/NextDenovo#83]
Hi，
  Ｉ am planning to use Nextdenovo for my nanopore reads correction , but it is unsure for me how much hard disk the file will take . As for my sequencing data ,its size is about 200G , so will 4T hard disk be enough for read correction ?  If not ,how much  should I prepare for it ?
  Thank you .

**Comment by moold** [comment]
Depending on the repeat content of the genome. But, generally speaking, It should be enough.

**Comment by longzhangnation** [comment]
In additon , for my correction ,the software stopped for several times , and I restart running it , will it rerun from where it stop ? Can you give me an estimate time about the correction of my data , my setting is 4 parallel * 6 threads .

**Comment by moold** [comment]
YES. It will rerun from where it stop. I cannot estimate the time, depending on your config and genome characteristic.
